kong/README.md

```diff
@@ -0,0 +1,51 @@
+Apache SkyWalking Nginx Agent For Kong
+==========
+
+This plugin base on Apache SkyWalking Nginx Agent for the Kong API gateway to integrate with the Apache SkyWalking distributed tracing system.
+
+## Usage
+
+1. Install the plugin on Kong:
+
+To install kong-plugin-skywalking:
+```bash
+$ luarocks install kong-plugin-skywalking --local
+```
+
+Edit kong.conf:
+```
+plugins = bundled,skywalking
+
+lua_package_path = ${user.home}/.luarocks/share/lua/5.1/?.lua;;
+```
+
+Set environment:
+```bash
+$ export KONG_NGINX_HTTP_LUA_SHARED_DICT="tracing_buffer 128m"
+```
+
+Restart Kong
+
+2. Enabling & configuring plugin:
+
+Add the plugin to an API:
+
+```bash
+$ curl -i -X POST \
+   --url http://localhost:8001/apis/{api_name}/plugins/ \
```

Comment:
`API` entity has been deprecated and removed in recent version of Kong, we can use `Services` here
```
--url http://localhost:8001/services/{service_name}/plugins/ \
```